include/swift/AST/Module.h

```diff
@@ -345,9 +345,17 @@ class ModuleDecl : public DeclContext, public TypeDecl {
 
   /// \sa getImportedModules
   enum class ImportFilter {
+    // Everything.
     All,
+
+    // @_exported only.
     Public,
-    Private
+
+    // Not @_exported only. Also includes @_usableFromInline.
```

Comment:
Hm, if these categories already aren't mutually exclusive, I think I'd suggest having UsableFromInline also include Public, as we discussed (which might suggest calling it "ForLinking" or something instead).

Reply:
Done